app/src/interfaces/file-image/file-image.vue

```diff
@@ -176,6 +175,16 @@ async function imageErrorHandler() {
 	}
 }
 
+function refreshImage() {
+	refresh();
+
+	loading.value = false;
+	lightboxActive.value = false;
+	editDrawerActive.value = false;
+	editImageDetails.value = false;
+	editImageEditor.value = false;
```

Comment:
Does this need to "reset" the image as well? Wouldn't just calling `refresh` be enough?

Reply:
Yes, you are right! Fixed it